src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -74,12 +77,12 @@ class Device(object):
         'vendor',
     ]
 
-    def __init__(self, path):
+    def __init__(self, path, lvs=[]):
```

Comment:
setting a mutable variable like this is a bit dangerous because the second time around this class gets instantiated it is not going to get a new list, but rather the previous one. This should illustrate the problem:

```

In [1]: class Foo(object): 
   ...:     def __init__(self, mutable=[]): 
   ...:         self.mutable = mutable 
   ...:                                                                                                                                                                                                            

In [2]: first = Foo()                                                                                                                                                                                              

In [3]: first.mutable                                                                                                                                                                                              
Out[3]: []

In [4]: first.mutable.append(1)                                                                                                                                                                                    

In [5]: first.mutable                                                                                                                                                                                              
Out[5]: [1]

In [6]: second = Foo()                                                                                                                                                                                             

In [7]: second.mutable                                                                                                                                                                                             
Out[7]: [1]
```

Reply:
So instead, it should be a `None` and then below it should be:

```python
self.lvs = lvs or []
```

Reply:
I've made the change.